test/modules/home/home_view_test.dart

```diff
@@ -238,5 +244,54 @@ void main() {
         expect(delegate.showDetailButtonDidTap, emits(surveys.first));
       });
     });
+
+    describe("it's showSideMenu() is called", () {
+      beforeEach((tester) async {
+        await tester.pumpAndSettle();
+        module.view.showSideMenu();
+        await tester.pumpAndSettle();
+      });
+
+      it("triggers delegate's sideMenuDidShow emits", (tester) async {
```

Comment:
triggers delegate sideMenuDidShow to emit - remember the last conversation we discussed